config.ini

```diff
@@ -1,6 +1,6 @@
 [SETTINGS]
 # Set the path to the location containing your certificates (root, private, claim certificate)
-SECURE_CERT_PATH = certs/
+SECURE_CERT_PATH = ./../../.aws_certificates/
 
```

Comment:
Good choice, but we're gonna need to make sure we always put all our stuff in the same directory

Reply:
will need a readme